timescaledb/how-to-guides/page-index/page-index.js

```diff
@@ -7,14 +7,24 @@ module.exports = [
       {
         title: "Install TimescaleDB",
         href: "install-timescaledb",
+        pageComponents: ['featured-cards'],
+        tags: ['timescaledb', 'install'],
+        keywords: ['install', 'TimescaleDB'],
+        excerpt: ['Install TimescaleDB'],
         children: [
           {
             title: "Timescale Forge",
-            href: "installation-forge"
+            href: "installation-forge",
+            tags: ['tsc', 'install'],
+            keywords: ['install', 'Timescale Cloud'],
+            excerpt: ['Install Timescale Forge']
           },
           {
             title: "Timescale Cloud",
-            href: "installation-cloud"
+            href: "installation-cloud",
+            tags: ['mst', 'install'],
+            keywords: ['install', 'Managed Service for TimescaleDB'],
```

Comment:
I think the team feels strongly we avoid using this verbiage until late September!

Reply:
Yep, I'll go through and revert that 👍